tasks/strings/strings.js

```diff
@@ -4,23 +4,40 @@
  * @param {string} str
  */
 function reverseString(str) {
+  if (typeof str !== "string") {
+    return 'This is not a string!';
+  }
+  const stringArr = str.split("");
+  stringArr.reverse();
+  return stringArr.join("");
```

Comment:
can be done in one-line:
return str.split('').reverse().join('');

Reply:
Yep, really. :)